index.md

```diff
@@ -8,31 +8,9 @@ layout: home
 {:.logo}
 ![Optech Logo](/img/logos/optech-notext.png)
 
-The Bitcoin Operations Technology Group (Optech) works to bring the best
-open source technologies and techniques to Bitcoin-using businesses in
-order to lower costs and improve customer experiences.
-
-An initial focus for the group is working with its member organizations to
-reduce transaction sizes and minimize the effect of subsequent transaction fee
-increases.  We provide [workshops][], [documentation][scaling book],
-[weekly newsletters][], [original research][dashboard], [case studies
-and announcements][blog], and help facilitate improved relations between
-businesses and the open source community.
-
-[scaling book]: https://github.com/bitcoinops/scaling-book
-[workshops]: /workshops
-[weekly newsletters]: /en/newsletters/
-[dashboard]: https://dashboard.bitcoinops.org/
-[blog]: /en/blog/
-
-Optech does not exist to make a profit, and all materials and documentation
-produced are released under the MIT license. We are supported by our generous
-founding sponsors and contributions from member companies.
-
-If you're an engineer or manager at a Bitcoin company or an open source contributor and you'd like to be a part of this, please
-contact us at [info@bitcoinops.org](mailto:info@bitcoinops.org).
-
-{% include newsletter-signup.html %}
+Bitcoin Optech works to bring the best open source technologies and techniques
```

Comment:
I think since this text appears on the about page, we could make the text even shorter and snappier on the index page. Something like:

"Bitcoin Optech helps Bitcoin users and businesses integrate scaling technologies. Learn more."

Thoughts? @harding - you raised a point about the word 'scaling' here: https://github.com/bitcoinops/bitcoinops.github.io/pull/117#issuecomment-470618659. Do you have any thoughts about its use here?

Reply:
If you think the term is appropriate, then we shouldn't stop using it.  In any case, we can always change this page again later if we move to a different phrasing, so there's no harm in this change.

Reply:
Thanks. I've updated the text in the front page.